<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDevice</class>
 <widget class="QWidget" name="SearchDevice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>设备ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_id"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>设备名称：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_name"/>
     </item>
     <item>
      <widget class="QPushButton" name="pdsearch">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbadddevice">
       <property name="text">
        <string>添加设备</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget_devices"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
